Tracker/tracker.lua

```diff
@@ -8916,13 +8916,15 @@ local function updateLoop()
         end
       
         -- Check where we need to move the play position
-        for i=1,#yloc-1 do
-          if ( ( gfx.mouse_y > yloc[i] ) and ( gfx.mouse_y < yloc[i+1] ) ) then
-            goto_position(i)
+        if tracker.cfg.pinPosition == 0 or tracker.lastleft == 0 then

```

Comment:
I'm not at my music PC right now, so from the top of my head - I recall having two issues with the row indicator:
* clicking and holding caused unexpected fast-forward/rewind behaviour because the underlying row number changes each time it's re-centered - so changed it to act only on a single click (no drag) when pinned
* when changing the fixed position with CTRA+ALT+LMB I didn't want it to change row also - so put a check to ensure that no modifiers were being held. This will have affected default behaviour in that previously holding, say, control and clicking would have changed position but now it is explicitly no modifiers. I hoped this wouldn't be an undesirable change in behaviour, but it is a change.

Reply:
Yeah, I think that change is perfectly acceptable. Also thanks for pointing it out already in the PR description.